server/handlers/project_command_output_handler.go

```diff
@@ -51,8 +57,13 @@ func (p *DefaultProjectCommandOutputHandler) Send(ctx models.ProjectCommandConte
 	}
 }
 
-func (p *DefaultProjectCommandOutputHandler) Receive(projectInfo string, callback func(msg string) error) error {
-	ch := p.addChan(projectInfo)
+func (p *DefaultProjectCommandOutputHandler) Receive(projectInfo string, ch chan string, callback func(msg string) error) error {
```

Comment:
ch is really not descriptive 😅 .  Maybe receiver?

Reply:
Done!